vscode-extension/src/extension.ts

```diff
@@ -780,19 +795,20 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
         `Error creating new file ${envTemplatePath}: ${err}`
       );
     }
+  }
 
-    const doc = await vscode.workspace.openTextDocument(envPath);
-    if (doc) {
-      vscode.window.showTextDocument(doc, {
-        preview: false,
-        // Tried using vscode.ViewColumn.Active but that overrides existing
-        // walkthrough window
-        viewColumn: vscode.ViewColumn.Beside,
-      });
-      vscode.window.showInformationMessage(
-        "Please define your environment variables."
-      );
-    }
+  // Open the env file that was either was created or already existed
+  const doc = await vscode.workspace.openTextDocument(envPath);
+  if (doc) {
+    vscode.window.showTextDocument(doc, {
+      preview: false,
+      // Tried using vscode.ViewColumn.Active but that overrides existing
+      // walkthrough window
+      viewColumn: vscode.ViewColumn.Beside,
+    });
+    vscode.window.showInformationMessage(
+      "Please define your environment variables."
+    );
```

Comment:
please add if/else to catch if this needed

Reply:
https://github.com/lastmile-ai/aiconfig/issues/1287